packages/table/src/layers/guides.tsx

```diff
@@ -52,7 +52,7 @@ export class GuideLayer extends React.Component<IGuideLayerProps, {}> {
             left: `${offset}px`,
         };
         const className = classNames(Classes.TABLE_OVERLAY, Classes.TABLE_VERTICAL_GUIDE, {
-            "pt-table-vertical-guide-flush-left": offset === 0,
+            [`${Classes.TABLE_VERTICAL_GUIDE}-flush-left`]: offset === 0,
```

Comment:
nit: since our classes have configurable PREFIX, then this will technically work, but it's worth noting this is slightly different that the other uses since we're building a classname string.